backend/copr_backend/storage.py

```diff
@@ -216,42 +217,62 @@ def init_project(self, dirname, chroot):
         response = self.client.create_publication(repository)
         return response.ok
 
-    def upload_build_results(self, chroot, results_dir, target_dir_name):
+    def upload_rpm(self, repository, path):
+        response = self.client.create_content(repository, path)
+
+        if not response.ok:
+            self.log.error("Failed to create Pulp content for: %s, %s",
+                           path, response.text)
+            return
```

Comment:
Linter is right; we should probably return failed response to be able to uniformly read it with others.

Reply:
I'll update it